Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1385,6 +1385,21 @@ public void RemovePlugin(IPlugin plugin)
             _plugins = list.ToArray();
         }
 
+        public IEnumerable<Assembly> GetApiPluginAssemblies()
+        {
+            var assemblies = _allConcreteTypes
+                .Where(i => typeof(ControllerBase).IsAssignableFrom(i))
+                .Select(i => i.Assembly)
+                .Distinct();
+
+            foreach (var assembly in assemblies)
```

Comment:
If you're iterating over it you have to `yield return`, otherwise you're evaluating it twice. You can either `yield return` here or move the debug logging to the service registration.